browser/src/view/components/Button/Button.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import './Button.scss';
+
+export const Button = ({ layout, label, Icon, type }) => {
+  return (
+    <button
+      className={`${
+        type === ButtonAttributes.Type.destructive
+          ? 'btn-destructive'
```

Comment:
Why did you decide to use `btn-` as a prefix?

Reply:
I think I was trying for added clarity, but I can see now how this prefix would be redundant given that 'destructive', etc. are only going to be a className property added to buttons

Reply:
Word.  2 things:
- unless someone specifically directs you to, try not to shorten words (btn vs button).  there's no reason to since we have compilers to do all we need to save memory and increase performance.
- if you do want a prefix, I'd make it more design system specific.  We don't really havea esign system but this is a common component.  We don't need to do this now though and might not ever.  YAGNI.